src/js/actions/ChaosActions.js

```diff
@@ -0,0 +1,104 @@
+var EventEmitter = require("events").EventEmitter;
+var lazy = require("lazy.js");
+var request = require("../helpers/qajaxWrapper");
+
+var config = require("../config/config");
+var util = require("../helpers/util");
+
+var AppsStore = require("../stores/AppsStore");
+var ChaosEvents = require("../events/ChaosEvents");
+
+function requestApp(appId, onSuccess, onError) {
+  request({
+    url: `${config.apiURL}v2/apps/${appId}`
+  })
+  .success(function (data) {
+    onSuccess(data.body.app);
+  })
+  .error(function () {
+    onError();
+  });
+}
+
+function deleteTasks(taskIds = [], onSuccess = util.noop) {
```

Comment:
@pierlo-upitup Yeah, your right optional arguments should have a default value.

But in this case, the onSuccess-callback here isn't optional. This default-value produces more harm then benefit, because it suppresses an error if no callback is given. It shouldn't have a default value.